CONTRIBUTING.md

```diff
@@ -47,47 +47,48 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The [demo project](demo) was initially forked from
-[dvc-checkpoints-mnist](https://github.com/iterative/dvc-checkpoints-mnist/tree/make_checkpoint),
-and is provided as a lightweight, convenient testbed to try the extension out
-with. It is not an exhaustive showcase of DVC's features, so testers are
-encouraged to try other DVC repositories -- especially real-world cases!
+The [demo project](demo) was initially forked from [dvc-checkpoints-mnist], and
+is provided as a lightweight, convenient testbed to try the extension out with.
+It is not an exhaustive showcase of DVC's features, so testers are encouraged to
+try other DVC repositories -- especially real-world cases!
 
-- Go to the `demo/` directory of this repo.
+- Install [Yarn](https://yarnpkg.com/) if needed. Go to the `demo/` directory of
+  this repo (`cd demo`) and run `yarn install` to set up the project.
 
-- It is recommended that you create the project's virtual environment with
-  Python 3.8 as some dependencies are not yet available for Python 3.9
+- Go back to the root of this repo (`cd ..`) and run `yarn setup:venv` to set up
+  a Python virtual environment in the demo project (in `demo/.env`).
```

Comment:
Does it work if you `yarn setup:venv` first from the root, and then `cd` into `demo/` for everything else? Instead of going back and forth in each step... ⏳